systemd/os-autoinst-openvswitch.service

```diff
@@ -10,7 +10,9 @@ Before=openqa-worker.target
 [Service]
 Type=dbus
 BusName=org.opensuse.os_autoinst.switch
+Environment=OS_AUTOINST_USE_BRIDGE=br0
```

Comment:
Ok, I wasn't sure about the ordering. But systemd.exec(5) states EnvironmentFile has priority so I rest my case :)